### DISCUSSION TITLE
how to run on local environment

### DISCUSSION THREAD

**Discussion by Roshan4518Git** [Edwardsoen/Leetcode-Premium-Unlocker#68] [General]
@Edwardsoen Please mention the steps to setup the project in local environment and steps to run the program

**Comment by Edwardsoen** [comment]
Sure, will write a small documentation to build locally

**Comment by ShlokJswl** [comment]
@Edwardsoen The extension has stopped working recently. Any idea why this is happening? Has it got anything to do with the updates on leetcode website??

**Comment by ayoubjoudii** [comment]
If there is an update to the extension or a problem annouce it please it doesn't work anymore

**Comment by HarshalShukla20** [comment]
It doesn't seem to work anymore.
